src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -1343,23 +1321,16 @@ class ContentSidebar extends PureComponent<Props, State> {
                             <Sidebar
                                 file={((file: any): BoxItem)}
                                 view={view}
+                                detailsSidebarProps={detailsSidebarProps}
                                 versions={versions}
                                 getPreviewer={getPreviewer}
                                 hasSkills={hasSkills}
                                 hasDetails={hasDetails}
-                                hasProperties={hasProperties}
                                 hasMetadata={hasMetadata}
-                                hasNotices={hasNotices}
-                                hasAccessStats={hasAccessStats}
-                                hasClassification={hasClassification}
                                 hasActivityFeed={hasActivityFeed}
                                 onDescriptionChange={this.onDescriptionChange}
                                 accessStats={accessStats}
-                                onAccessStatsClick={onAccessStatsClick}
-                                onClassificationClick={onClassificationClick}
-                                onVersionHistoryClick={onVersionHistoryClick}
                                 onSkillChange={this.onSkillChange}
-                                hasVersions={hasVersions}
                                 accessStatsError={accessStatsError}
```

Comment:
don't you want to move the errors related to the details sidebar in there too?
As well as the onDescriptionChange.

You can merge these internal ones into detailsSidebarProps like

detailsSidebarProps={{
 foo: bar,
...detailsSidebarProps
}}

Reply:
good call.